### PR TITLE
Toha customzing layout

### PR DESCRIPTION
**Pull request by Rihoj**
I hope this post is acceptable and well received. I found I had a little more trouble than I liked getting started since I was new to Hugo.

### CODE REVIEW

File: content/posts/customizing/customize-layout/index.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+menu:
+  sidebar:
+    name: Layout Customization
+    identifier: customizing-layout
+    parent: customizing
+    weight: 411
+---
+
+## We've all been there
+We all want to be able to customize the layout of a given theme, and I with this short tutorial I will show you how. I stumbled my way through this, and I hope you won't have to as well.
```
Comment (hossainemruz) [inline content/posts/customizing/customize-layout/index.md:17]: Please, change `and I with this` to `and in this`

**Review by hossainemruz — CHANGES_REQUESTED** [review]
Hi @Rihoj . Thank you for this short but effective tutorial. I really appreciate it.

I have requested a small change. Can you please acknowledge it?

File: content/posts/customizing/customize-layout/index.md (hunk 1 above)
Comment (Rihoj) [reply content/posts/customizing/customize-layout/index.md]: Thank you for that catch!

**Review by hossainemruz — APPROVED** [review]
Looks good now.

**Pull request merged**